ydb/public/sdk/cpp/client/ydb_scheme/scheme.h

```diff
@@ -49,8 +49,7 @@ enum class ESchemeEntryType : i32 {
     ExternalTable = 18,
     ExternalDataSource = 19,
     View = 20,
-    ResourcePool = 21,
-    Transfer = 23,
+    ResourcePool = 21
```

Comment:
```suggestion
    ResourcePool = 21,
```
Чтобы совсем не было диффа.